addons/web_tour/static/src/scss/tip.scss

```diff
@@ -238,3 +238,8 @@ $o-tip-size-delay-in: $o-tip-duration-in - $o-tip-size-duration-in;
         display: none !important;
     }
 }
+
+.o_center_on_text {
+    opacity: 0;
+    position: absolute;
+}
```

Comment:
I think it is a good practice to end all files with an empty line. See:
https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file